docs/security.rst

```diff
@@ -142,6 +142,32 @@ Now you can continue using urllib3 as you normally would.
 
 For more details, check the :mod:`~urllib3.contrib.pyopenssl` module.
 
+Installing urllib3 with SNI support and certificates
```

Comment:
It's kinda redundant with the section just above, should it be a subsection? Or hmm.. Need somekind of restructuring of this doc.